10-Modelling-Deformable-Interfaces-with-Meshes/02-Vesicle-MD.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>I would rephrase the 2nd sentence: "Pass the <code>mesh_obj</code> to the neighborlist and set the <code>"meshbond"</code> exclusion to exclude WCA forces between vertices in the same triangle".</p><p> </p><p>Also, does this really exclude all vertices in the same triangle? Or only those that form an edge?</p><p> </p><p>Lastly, WCA has not previously been defined in the tutorials. You could replace "WCA" with "repulsive".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>